cdk/domino_cdk/config/eks.py

```diff
@@ -13,6 +16,7 @@ class EKS:
     max_nodegroup_azs: 3 - Will provision nodegroups in up to this many availability zones.
     global_node_labels: some-label: "true"  - Labels to apply to all kubernetes nodes
     global_node_tags: some-tags: "true"  - Labels to apply to all kubernetes nodes
+    secrets_encryption_key_arn: None  - KMS key name/arn to encrypt kubernetes secrets
```

Comment:
Should mention that if this is `None` we'll create the key, as this comment will get added to the generated config file (which will be replacing the static `config_template.yaml`).